openlibrary/core/vendors.py

```diff
@@ -14,6 +14,7 @@
 
 
 BETTERWORLDBOOKS_API_URL = 'http://products.betterworldbooks.com/service.aspx?ItemId='
+amazon_full_date = re.compile('\d{4}-\d\d-\d\d')
```

Comment:
- uppercase for consistency with the other constant in this file
- suffix with `_RE` because otherwise the name makes it seem like a date object when viewed from this context

```suggestion
AMAZON_FULL_DATE_RE = re.compile('\d{4}-\d\d-\d\d')
```